toonz/sources/toonz/startuppopup.cpp

```diff
@@ -409,7 +410,13 @@ void StartupPopup::refreshRecentScenes() {
     int i = 0;
     for (QString name : m_sceneNames) {
       if (i > 9) break;  // box can hold 10 scenes
-      QString justName = QString::fromStdString(TFilePath(name).getName());
+      QString justName    = QString::fromStdString(TFilePath(name).getName());
+      QString projectName = QString::fromStdString(
+          TProjectManager::instance()
+              ->getProjectNameByScenePath(TFilePath(name))
+              .getName());
+      if (projectName.length() > 0)
+        justName             = justName + " - " + projectName;
       m_recentNamesLabels[i] = new StartupLabel(justName, this, i);
       m_recentNamesLabels[i]->setToolTip(
           name.remove(0, name.indexOf(" ") + 1));  // remove "#. " prefix
```

Comment:
This PR introduces the way to get the file name list without number prefixes. So you can use `name` for the tooltip instead of `name.remove(0, name.indexOf(" ") + 1)` .